db/migrations/202407012100_transactions.go

```diff
@@ -0,0 +1,59 @@
+package migrations
+
+import (
+	_ "embed"
+
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// This migration
+// - Replaces the old payments table with a new transactions table
+// - Adds new properties to apps
+//   - isolated boolean
+//
+// - Renames max amount on app permissions to be clear its in sats
+var _202407012100_transactions = &gormigrate.Migration{
+	ID: "202407012100_transactions",
+	Migrate: func(tx *gorm.DB) error {
+
+		if err := tx.Exec(`
+CREATE TABLE transactions(
```

Comment:
transactions is also often a reserved word - esp in the DB context this can lead to strange things. are we good there?

Reply:
it works, but I also don't like the same terminology being used for 2 different things. I added a note to https://github.com/getAlby/hub/issues/299